client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -171,8 +171,8 @@ class JetpackSyncPanel extends React.Component {
 				<div className="jetpack-sync-panel__action">
 					{ translate(
 						'Jetpack Sync keeps your WordPress.com dashboard up to date. ' +
-							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience. '
-					) }
+							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience.'
+					) + ' ' }
```

Comment:
Should this space be optionally displayed only if the text afterwards is being displayed too?